modules/core/src/test/java/org/apache/ignite/testframework/GridTestUtils.java

```diff
@@ -392,6 +384,50 @@ public static void assertMatches(@Nullable IgniteLogger log, String str, String
         }
     }
 
+    /**
+     * Checks that file contains line {@param str} matches given regular expression {@param regexp}. Logs both strings
+     * and throws {@link java.lang.AssertionError}, if not.
+     *
+     * @param log Logger (optional).
+     * @param filePath Absolute path to file.
+     * @param charset File charset.
+     * @param regexp Regular expression pattern.
+     */
+    public static void assertMatchesLine(@Nullable IgniteLogger log, String filePath, String charset, String regexp) throws IOException {
+        assertMatchesLine(log, filePath, charset, Pattern.compile(regexp));
+    }
+
+    /**
+     * Checks that file contains line {@param str} matches given regular expression {@param regexp}. Logs both strings
+     * and throws {@link java.lang.AssertionError}, if not.
+     *
+     * @param log Logger (optional).
+     * @param filePath Absolute path to file.
+     * @param charset File charset.
+     * @param regexp Regular expression pattern.
+     */
+    public static void assertMatchesLine(@Nullable IgniteLogger log, String filePath, String charset, Pattern regexp) throws IOException {
+        try (Scanner input = new Scanner(new FileInputStream(filePath), charset)) {
+            input.useDelimiter("[\\s]*\\n[\\s]*");
+
+            boolean found = false;
+
+            while (input.hasNext()) {
+                if (found = input.hasNext(regexp))
+                    break;
+                else
+                    input.nextLine();
+            }
+
+            if (!found) {
+                U.warn(log, String.format("File doesn't does not contain line matches regexp: '%s'", regexp));
```

Comment:
Please fix grammar in the sentence. The same applies to the next line.

Reply:
Done